app/models/provider.rb

```diff
@@ -110,6 +110,7 @@ def current_accredited_courses
   scope :search_by_code_or_name, ->(search_term) {
     where("provider_name ILIKE ? OR provider_code ILIKE ?", "%#{search_term}%", "%#{search_term}%")
   }
+  scope :with_full_name, ->(provider_name) { where(provider_name: provider_name) }
```

Comment:
For the sake of discussion, is there a benefit over just using `where(provider_name: provider_name)`? As I see it:

Pros:
- a few characters less
- abstracts away how we find the provider name

Cons:
- one more line of code in the model
- ~20 lines of specs

Are the Pros outweighing the Cons here?

Reply:
If `with_full_name` is part of a chain of scopes then perhaps stylistically, nice to have at same level of abstraction otherwise not sure if there are many advantages